CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# How to contribute
+
+Third-party patches are essential for keeping vanagon great. We simply can't
+access the huge number of platforms and myriad configurations for running
+vanagon. We want to keep it as easy as possible to contribute changes that
+get things working in your environment. There are a few guidelines that we
+need contributors to follow so that we can have a chance of keeping on
+top of things.
+
+## Getting Started
+
+* Make sure you have a [Jira account](http://tickets.puppetlabs.com)
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* Submit a ticket for your issue, assuming one does not already exist.
```

Comment:
For now, yes, I used CPR in the example ticket below, but calling it out more specifically would probably be good.